src/ctest.c

```diff
@@ -575,10 +575,12 @@ CTEST_DEFINE_EQUALITY_ASSERTION_FUNCTIONS_FOR_TYPE(int64_t,)
 static char* ctest_vsprintf_char(const char* format, va_list va)
 {
     char* result;
-    int neededSize = vsnprintf(NULL, 0, format, va);
+    va_list va_clone;
+    va_copy(va_clone, va);  /*this is needed because "va" would be passed twice to a function that parses it with va_arg: once in vsnprintf(NULL, 0, format, va) and the second time in vsnprintf(result, neededSize + 1, format, va) yielding undefined behavior*/
```

Comment:
>va_copy [](start = 4, length = 7)

This is missing va_end. I did not pay attention to the previous review, but that might be the case there too.

Reply:
Such is the price. Sigh. Will look back and here (again). Thanks!

Reply:
I've looked at testrunnerswitcher from before and It had va_end. Phew!